src/app/recipe-detail-page/recipe-detail-page.component.ts

```diff
@@ -1,28 +1,315 @@
 import { Component, OnInit } from '@angular/core';
 import { MatInputModule } from '@angular/material/input';
 import { MatFormFieldModule, MatFormField, MatLabel } from '@angular/material/form-field';
+import {MatListModule} from '@angular/material/list';
 import {FormBuilder, FormGroup} from '@angular/forms';
 import {FormControl, Validators, ReactiveFormsModule} from '@angular/forms';
+import {MatDialog} from '@angular/material/dialog';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
import {MatDialog} from "@angular/material/dialog";
```